src/components/AddCharacterForm.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  useDisclosure,
+  VStack,
+  Input,
+  FormControl,
+  HStack,
+  WrapItem,
+  Button,
+  ModalCloseButton,
+} from '@chakra-ui/react';
+import ButtonTemp from './Button';
+import FormInput from './FormInput';
+
+const AddCharacterForm = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <ButtonTemp onClick={onOpen} buttonText="Add a Character" />
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent bg={'black'}>
+          <ModalHeader textColor={'white'}>Add a Character</ModalHeader>
```

Comment:
Something else I forgot to comment on earlier is to try to stick to using double quotes for string attributes instead of single quotes inside curly braces (so `bg={'black'}` should be `bg="black"`)

Reply:
Also make sure to remove unused imports!